sci-libs/vtk/files/vtk-9.0.1-0001-fix-kepler-compute_arch-if-CUDA-toolkit-11-is-used.patch

```diff
@@ -0,0 +1,32 @@
+From 33d9b7ded14ce36e3f63810f3403623ee5c2059c Mon Sep 17 00:00:00 2001
```

Comment:
Patch links etc if any too, but okay if not, at the top of the [bug](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) (same for all patches)

Reply:
Those patches are from me and reflect the commit-id that was created when I commited the changes. I can remove the line, if the patch still works without it, but have never done this so far.